### ISSUE TITLE
Show recommended badge on search results page

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-frontend#7793]
UX is as follows:

![Screenshots](https://user-images.githubusercontent.com/1514/55243385-07db4400-5237-11e9-897b-8b90047c88a4.jpg)

Depends on changes in https://github.com/orgs/mozilla/projects/68

See https://mozilla.invisionapp.com/share/BXQTIIC86ZN for the latest UX

**Comment by bobsilverberg** [comment]
@jvillalobos Do we want to show recommended status (e.g., a tiny badge) in autocomplete results (i.e., what you see when you use the search box - see example below).

![Screenshot 2019-04-29 13 25 24](https://user-images.githubusercontent.com/142755/56914527-52dcb700-6a82-11e9-802d-bab71932087f.png)

**Comment by jvillalobos** [comment]
It'd be nice to show the badge there, yes. @brassy-, thoughts?

**Comment by bobsilverberg** [comment]
Here's the most recent InVision prototype for the search results:

<img width="704" alt="Screenshot 2019-05-07 14 38 16" src="https://user-images.githubusercontent.com/142755/57324422-c65f7380-70d5-11e9-84c3-e2edb14aa0fc.png">

**Comment by brassy-** [comment]
> It'd be nice to show the badge there, yes. @brassy-, thoughts?

Considering how condesended the list, I recommend to the badge at opposite of the name
Like this:
![image](https://user-images.githubusercontent.com/6309827/57457773-6c46e580-7270-11e9-80e5-63ea51101171.png)

**Comment by willdurand** [comment]
@brassy- 👋

We would have to hide the recommended icon when the mouse hovers a suggested item because, on hover/active/focus, we render a little arrow:

![Screen Shot 2019-05-09 at 16 18 50](https://user-images.githubusercontent.com/217628/57460884-3e649f80-7276-11e9-83ec-2c7fd6a264fb.png)

Would that work for you?

**Comment by brassy-** [comment]
> @brassy- 👋
> 
> We would have to hide the recommended icon when the mouse hovers a suggested item because, on hover/active/focus, we render a little arrow:
> 
> ![Screen Shot 2019-05-09 at 16 18 50](https://user-images.githubusercontent.com/217628/57460884-3e649f80-7276-11e9-83ec-2c7fd6a264fb.png)
> 
> Would that work for you?

It works for me — and it creates a sort of animation which can help catching attention.

**Comment by ioanarusiczki** [comment]
@bobsilverberg I verified this issue on AMO dev - FF67(Win10). 
- the badge shows for all recommended add-on types, I've tested around for search results (with and without a query to see them ordered correctly) . 
- the entire badge area is displayed with a 16px height and 96.93 px length. The badge is 16x16 px.
- the small screen issue was https://github.com/mozilla/addons-frontend/issues/8040
- https://github.com/mozilla/addons-frontend/issues/8067 was filed because the badge should not show up for Android, so right now I'll be focusing on the Android  search behavior on AMO dev.
